### ISSUE TITLE
Crash On Hovering Over A Enchanted Pick

### ISSUE THREAD

**Issue by Cyranos** [Dplayend/Toggle-Enchantments#7]
Kinda per title, I opened a barrel which had a iron pickaxe in it enchanted with Efficiency 2 & Unbreaking 1 which caused a crash. When I disabled Toggle Enchants & loaded the game I was able to re-open the barrel and hover over the pickaxe perfectly fine and was able to put it in my backpack from sophisticated backpacks. Though if I re-enable the mod it crash upon hovering over the pickaxe. 

Mc Version: 1.20.1
Forge Version: 47.2.1
Toggle Enchants Version: 1.20.x-v1.0

Crash Log;
[Here](https://mclo.gs/aDDi14b)

**Comment by Ch4mp10** [comment]
I've had the same issue with the 1.19.2 fabric version of the mod as well. Happens with a gold sword w/ knockback II.
Crash Log:
[crash-2023-11-02_09.58.15-client.txt](https://github.com/Dplayend/Toggle-Enchantments/files/13239741/crash-2023-11-02_09.58.15-client.txt)

**Comment by DarKnight594** [comment]
Do you have a mod called sophisticated storage? Because some other people have problem with it too with Toggle Enchants

**Comment by Ch4mp10** [comment]
No, not that mod. I do have Tom's Simple Storage (though it is in a regular double chest) and Mouse Wheelie. Not sure if those might be interfering.

**Comment by DarKnight594** [comment]
I am not sure but I think it might be. Try removing the storage mod and play. If problem still occurs then wait for the developer to say anything on this

**Comment by Traptonian** [comment]
having this problem too i also have toms storage
